Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -620,10 +617,10 @@ theorem symm_trans_mem_contDiffGroupoid (e : PartialHomeomorph M H) :
 
```

Comment:
About line 550, can't comment directly there: please change "Smooth functions" to "C^n"